src/main/java/com/epam/reportportal/auth/integration/builder/LdapBuilder.java

```diff
@@ -30,11 +27,8 @@ public LdapBuilder(LdapConfig ldapConfig) {
 
 	public LdapBuilder addUpdateRq(UpdateLdapRQ updateLdapRQ) {
 		ldapConfig.setEnabled(updateLdapRQ.getLdapAttributes().getEnabled());
-		ldapConfig.setPasswordEncoderType(PasswordEncoderType.findByType(updateLdapRQ.getPasswordEncoderType())
-				.orElseThrow(() -> new ReportPortalException(
-						ErrorType.BAD_REQUEST_ERROR,
-						Suppliers.formattedSupplier("Unsupported password encoder type - {}", updateLdapRQ.getPasswordEncoderType()).get()
-				)));
+
+		PasswordEncoderType.findByType(updateLdapRQ.getPasswordEncoderType()).ifPresent(ldapConfig::setPasswordEncoderType);
```

Comment:
what would be if password encoder type provided with update ldap rq doesn't presents?

Reply:
if password encoder type wasn't provided Ldap password attribute wouldn't be encrypted